<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit_3">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
    border: 1px solid #282c34;  /* 蓝色边框 */
    border-radius: 5px;         /* 圆角 */
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    /*background-color: #282c34;  /* 背景色 */
    color: black;               /* 字体颜色 */
    border: 1px solid #282c34;  /* 边框 */
    border-radius: 5px;         /* 圆角 */
}

QPushButton:hover { 
    /*background-color: #3b414d;  /* 鼠标悬停时的背景色 */
	font-weight: bold;
}

QPushButton:pressed {
    background-color: #e5e5e5;  /* 按下时的背景色 */
}</string>
        </property>
        <property name="text">
         <string>开   始</string>
        </property>
        <property name="iconSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    /*background-color: #282c34;  /* 背景色 */
    color: black;               /* 字体颜色 */
    border: 1px solid #282c34;  /* 边框 */
    border-radius: 5px;         /* 圆角 */
}

QPushButton:hover { 
    /*background-color: #3b414d;  /* 鼠标悬停时的背景色 */
	font-weight: bold;
}

QPushButton:pressed {
    background-color: #e5e5e5;  /* 按下时的背景色 */
}</string>
        </property>
        <property name="text">
         <string>保   存</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::WeatherShowers"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>110</y>
         <width>132</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入时间：</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>21</y>
         <width>240</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>345</x>
         <y>21</y>
         <width>79</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>MONKEY</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>99</x>
         <y>21</y>
         <width>240</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>21</y>
         <width>59</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>MTBF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>345</x>
         <y>110</y>
         <width>72</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>任务开始时间</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>99</x>
         <y>110</y>
         <width>132</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>110</y>
         <width>72</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>任务开始时间</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
